xtask/src/install.rs

```diff
@@ -33,6 +34,16 @@ impl InstallCmd {
         if let Some(client) = self.client {
             install_client(client).context("install client")?;
         }
+        if both {
+            eprintln!(
```

Comment:
@matklad, what is the reason to print to `stderr` and not `stdout`?

Reply:
stderr is for humans, stdout is for tools. The `err` in `stderr` is slightly misleading.